### DISCUSSION TITLE
PWA manifest.json

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#16339] [Q&A]
Does someone has experience with adding a manifest.json file to your Orchard Core website? I tried to add it but it's not recognized when I try to add the website to the homescreen of my phone.

**Comment by Piedone** [comment]
I think, but I'm not sure, that you can add this file to the wwwroot folder of your root web app and it'll be served under /manifest.json there. This would be for the whole app though, the same for all tenants.

A slightly better approach would be to do what the SEO module does, which returns a robots.txt file [from a middleware](https://github.com/OrchardCMS/OrchardCore/blob/7a14fd9bca4e667140c3db93753f82cb61e93ed5/src/OrchardCore.Modules/OrchardCore.Seo/Services/RobotsMiddleware.cs#L12).

**Comment by ns8482e** [comment]
Nice to have PWA manifest as feature
